packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/dq-response.ts

```diff
@@ -38,7 +38,21 @@ function getHomeCommunityIdForDr(extrinsicObject: ExtrinsicObject): string {
   return getResponseHomeCommunityId(extrinsicObject);
 }
 
-function getCreationTime(time: string | undefined): string | undefined {
+function getCreationTime({
+  creationTimeValue,
+  serviceStartTimeValue,
+  serviceStopTimeValue,
+}: {
+  creationTimeValue: string | undefined;
```

Comment:
Your saying its a string or undefined here so then why do you need to do `String(creationTimeValue)` below